tests/test_zenodo_crawler.py

```diff
@@ -39,7 +39,8 @@ def mock_zenodo_query():
                             "links": {
                                 "self": "https://www.test.file.com/nofilehere"
                             },
-                            "type": "json"
+                            "type": "json",
+                            "size": 45346
```

Comment:
Should this be a fixed number - even in a mocked test?  I'd feel better if this were computed somehow so that you don't have to remember to update this if the test changes.

Reply:
I don't think we need to have this computed since the file size will always be available via Zenodo api